templates/projects/view.tmpl

```diff
@@ -1,263 +1,192 @@
 						</div>
-					</div>
-				{{end}}
-			</div>
-			<div class="divider"></div>
-
-			<div class="ui cards board" data-url="{{$.Link}}/{{.ID}}" data-project="{{$.Project.ID}}" data-board="{{.ID}}" id="board_{{.ID}}">
-
-				{{range (index $.IssuesMap .ID)}}
-
-				<!-- start issue card -->
-				<div class="card board-card" data-issue="{{.ID}}">
-					{{if eq $.Project.CardType 1}}{{/* Images and Text*/}}
-						<div class="card-attachment-images">
-							{{range (index $.issuesAttachmentMap .ID)}}
-								<img src="{{.DownloadURL}}" alt="{{.Name}}" />
-							{{end}}
-						</div>
 					{{end}}
-					<div class="content gt-p-0">
-						<div class="header">
-							<span class="gt-dif gt-ac gt-vm">
-								{{template "shared/issueicon" .}}
-							</span>
-							<a class="project-board-title gt-vm" href="{{.Link}}">
-								{{.Title}}
-							</a>
-						</div>
-						<div class="meta gt-my-2">
-							<span class="text light grey">
-								{{.Repo.FullName}}#{{.Index}}
-								{{$timeStr := TimeSinceUnix .GetLastEventTimestamp $.locale}}
-								{{if .OriginalAuthor}}
-									{{$.locale.Tr .GetLastEventLabelFake $timeStr (.OriginalAuthor|Escape) | Safe}}
-								{{else if gt .Poster.ID 0}}
-									{{$.locale.Tr .GetLastEventLabel $timeStr (.Poster.HomeLink|Escape) (.Poster.GetDisplayName | Escape) | Safe}}
-								{{else}}
-									{{$.locale.Tr .GetLastEventLabelFake $timeStr (.Poster.GetDisplayName | Escape) | Safe}}
-								{{end}}
-							</span>
-						</div>
-						{{- if .MilestoneID}}
-						<div class="meta gt-my-2">
-							<a class="milestone" href="{{$.RepoLink}}/milestone/{{.MilestoneID}}">
-								{{svg "octicon-milestone" 16 "gt-mr-2 gt-vm"}}
-								<span class="gt-vm">{{.Milestone.Name}}</span>
-							</a>
-						</div>
-						{{- end}}
-						{{- range index $.LinkedPRs .ID}}
-						<div class="meta gt-my-2">
-							<a href="{{$.RepoLink}}/pulls/{{.Index}}">
-								<span class="gt-m-0 text {{if .PullRequest.HasMerged}}purple{{else if .IsClosed}}red{{else}}green{{end}}">{{svg "octicon-git-merge" 16 "gt-mr-2 gt-vm"}}</span>
-								<span class="gt-vm">{{.Title}} <span class="text light grey">#{{.Index}}</span></span>
-							</a>
-						</div>
-						{{- end}}
-					</div>
+				</div>
 
-					{{if or .Labels .Assignees}}
-					<div class="extra content labels-list gt-p-0 gt-pt-2">
-						{{range .Labels}}
-							<a target="_blank" href="{{$.RepoLink}}/issues?labels={{.ID}}">{{RenderLabel $.Context .}}</a>
-						{{end}}
-						<div class="right floated">
-							{{range .Assignees}}
-								<a target="_blank" href="{{.HomeLink}}" data-tooltip-content="{{$.locale.Tr "repo.projects.column.assigned_to"}} {{.Name}}">{{ctx.AvatarUtils.Avatar . 28 "mini gt-mr-3"}}</a>
-							{{end}}
+				<div class="divider"></div>
+
+				<div class="ui cards" data-url="{{$.Link}}/{{.ID}}" {{if $.CanWriteProjects}}style="cursor: grab;"{{end}} data-project="{{$.Project.ID}}" data-board="{{.ID}}" id="board_{{.ID}}">
```

Comment:
Wouldn't mind adding a [`gt-cursor-grab`](https://tailwindcss.com/docs/cursor) in place of this inline style.